packages/create-novu-app/README.md

```diff
@@ -1,7 +1,27 @@
+<div align="center">
+  <a href="https://novu.co?utm_source=github" target="_blank">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/2233092/213641039-220ac15f-f367-4d13-9eaf-56e79433b8c1.png">
+    <img alt="Novu Logo" src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280"/>
+  </picture>
+  </a>
+</div>
+
 # Create Novu App
 
+[![Version](https://img.shields.io/npm/v/create-novu-app.svg)](https://www.npmjs.org/package/create-novu-app)
+[![Downloads](https://img.shields.io/npm/dm/create-novu-app.svg)](https://www.npmjs.com/package/create-novu-app)
+
 The easiest way to get started with Novu workflows is to call `create-novu-app`. The command will ask you for a project name as well as if you want to use react email.
 
-### Why use Create Novu App?
+## Quickstart
+
+You can get started by retrieving your Novu API key from the [settings page](https://web.novu.co/settings), then run:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** ‏ The url will navigate the users to the US version of the site, but the EU users might get confused, don't know if we care a this point.

Reply:
EU takes about 1% of the traffic that US has today, EU users will know to add the `eu.` prefix. We don't care right now.